### PR TITLE
Navbar - BrickHack Club Site

### LINKED ISSUE #1207: Navbar - BrickHack Club Site
Make navbar for #1206 

[Figma](https://www.figma.com/file/ZR9Mb4xNRrnoRZoX0bCWsX/BrickHack-Club-Website?node-id=0%3A1)

This issue will also contain the site's file creation and basic sectioning outline

### PR DESCRIPTION
**Pull request by sjmiller7**
Fixes #1207 and adds general file structure + styles for the site

### CODE REVIEW

**Comment by sjmiller7** [comment]
I don't _have_ to make it close the nav when i click out, right? right??

**Comment by sjmiller7** [comment]
The way I have the mobile navbar set up, it currently moves from fixed when open to static when closed. This means that when it opens/closes, the content jumps around a bit. I don't _think_ this is an issue but others may feel differently.

File: bhclub.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        <div>
+            <a class="btn-light" href="https://discord.gg/gNJXjvzkuB" target="_blank">JOIN</a>
+            <a href="#about" class="nav-link">About</a>
+            <a href="#events" class="nav-link">Events</a>
+            <a href="#teams" class="nav-link">Teams</a>
+            <a href="#contact" class="nav-link">Contact</a>
+        </div>
+        <div>
+            <a class="social-link" href="https://discord.gg/gNJXjvzkuB" target="_blank">Discord</a>
+            <a class="social-link" href="#" target="_blank">Github</a>
```
Comment (peterkos) [inline bhclub.html:35]: nit: `GitHub`, not `Github`

File: bhclub.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    $('main').removeClass('main-pointer');
+});
+
+// Closing the navbar when outside of the nav is clicked
+$(document).on('click', 'main', function() {
+    $('nav').removeClass('show-nav');
+    $('#toggle').removeClass('fa-times');
+    $('#toggle').addClass('fa-bars');
+    $('main').removeClass('main-pointer');
+});
```
Comment (peterkos) [inline bhclub.js:34]: nit: newline EOF

VSCode should have a setting to auto-add this btw

File: index.html [hunk 3]
```diff
@@ -78,7 +78,8 @@
     </section>
     <footer>
         <p>Interested in sponsoring? <span class="text-break">E-mail <a href="mailto:sponsorship@brickhack.io">sponsorship@brickhack.io</a></span></p>
-        <p>View the <a href="./bh7.html">BrickHack7 website</a></p>
+        <p>Want to help organize? <span class="text-break"><a href="./bhclub.html">BrickHack Club website</a></span></p>
+        <p>View the <a href="./bh7.html">Meet the team</a></p>
```
Comment (peterkos) [inline index.html:82]: `View the meet the team`

**Review by peterkos — COMMENTED** [review]
Looks great! A couple nits and a couple actual changes

**Pull request merged**